### ISSUE TITLE
Deprecate finger (Was: Move finger service to finger.peeringdb.com)

### ISSUE THREAD

**Issue by job** [peeringdb/peeringdb#264]
We currently operate a FINGER service on port 79 on peeringdb.{com/org/net} and *.peeringdb.{com/org/net}. By having the finger service on the same domain names as the HTTPS services, we hinder improvements on the HTTPS service.

I propose that the FINGER service is moved to `finger.peeringdb.com` which shall be the canonical name for the finger service. 

The old finger service on *.peeringdb.com and peeringdb.com should be updated and show as preamble & postamble to the finger output something like:

> % Per X/Y/2018 the finger service will only be available at finger.peeringdb.com

Then, after 4 to 6 weeks, the finger service on *.peeringdb.com and peeringdb.com can be turned off.

**Comment by eloos** [comment]
change of hosting domain, no code change, see #263

**Comment by mahtin** [comment]
I would vote to deprecate ``finger`` because ... well because it's a frigging old protocol. Sometimes you need to move on! Plus ... what's the usage level on ``finger`` vs. ``whois`` (which I don't presently recommend deprecating.

Plus, as ``finger`` doesn't work on users or usernames (which is what RFC742 & [RFC1288](https://tools.ietf.org/html/rfc1288) was built for), I don't see it's value anymore.
```
$ echo mahtin | nc finger.peeringdb.com 79
$
$ echo mahtin@mahtin.com | nc finger.peeringdb.com 79
$
```
It does work for AS#'s (which is better served by ``whois``).

**Comment by mcmanuss8** [comment]
Vote:
1) Deprecate
2) Move to different hostname

I vote 1

**Comment by grizz** [comment]
vote 1

Want to clarify, we 're removing it and not deprecating for removal at a later period, correct?

**Comment by job** [comment]
For finger, my take would be that we just announce our intention via email as a change in some upcoming release. I wouldn't want to ask the vendor to do any work, other than removing it.

**Comment by grizz** [comment]
That's something pdb-ops could do easily, I don't think this needs to go through a vendor.

**Comment by job** [comment]
ack. question still stands for PC whether we want to remove it or not, regardless of who executes.

Thanks for pointing out OPS can do it

**Comment by arnoldnipper** [comment]
remove it

**Comment by eloos** [comment]
Vote 1, remove

**Comment by job** [comment]
6 votes in favor of deprecating the `finger` service, no opposition. 

I'd like to suggest we sunset this service on September 15th 2018.

I can email out announcements in the next few days.

**Comment by job** [comment]
small note: PDB OPS will execute this work.

**Comment by eloos** [comment]
vote 1 remove, no code-change until a later clean-up

**Comment by grizz** [comment]
@job agreed, notifications should go out (cc @eloos)

**Comment by ccaputo** [comment]
Per http://lists.peeringdb.com/pipermail/pdb-announce/2018-November/000113.html the finger service is no longer reachable.  The load balancers have had it disabled and port 79 has been blocked on the ACLs.

The code is still operational on the servers.  If that code should be disabled/removed, please re-open and assign this to a developer, or create a new issue for it.
